channel_test.go

```diff
@@ -78,6 +78,29 @@ func TestClient_CreateChannel(t *testing.T) {
 	}
 }
 
+func TestChannel_GetManyMessages(t *testing.T) {
+	ctx := context.Background()
+	c := initClient(t)
+	userA := randomUser(t, c)
+	userB := randomUser(t, c)
+
+	ch := initChannel(t, c, userA.ID, userB.ID)
+	resp1, err := c.CreateChannel(ctx, ch.Type, ch.ID, userA.ID, nil)
```

Comment:
not sure to understand why are we calling CreateChannel here?

Reply:
no clue either 😅 fixed it